test/e2e/far_e2e_test.go

```diff
@@ -39,12 +69,64 @@ var _ = Describe("FAR E2e", func() {
 			Expect(k8sClient.Get(context.Background(), client.ObjectKeyFromObject(far), testFarCR)).To(Succeed(), "failed to get FAR CR")
 		})
 	})
+
+	Context("fence agent - non-Dummy", func() {
```

Comment:
maybe a better name than `non-Dummy` ?

Reply:
I was using the term `dummy` for FA which is dumb and does not act as a real FA in comparison to a real one who
supports some [FA actions](https://github.com/ClusterLabs/fence-agents/blob/main/doc/FenceAgentAPI.md#agent-operations-and-return-values) and actually communicate with a machine/node.

Is `fence agent - fence_aws or fence_ipmilan` sounds better to you?
Anyway will address this possible change in #20 and merge the current PR.